docs/manual_curation.md

```diff
@@ -0,0 +1,111 @@
+# Manual trait curation
+
+The goal is for traits with occurence ≥ 10 to have 100% coverage after the manual curation. For the rest of the
+traits, curate as many as possible.
+
+## Extract information about previous mappings
+At this step, mappings produced by the pipeline on the previous iteration (including automated and manual) are
+downloaded to be used to aid the manual curation process.
+
+```bash
+# Download the latest eva_clinvar release from FTP
+wget -qO- ftp://ftp.ebi.ac.uk/pub/databases/eva/ClinVar/latest/eva_clinvar.txt \
+  | cut -f4-5 | sort -u > previous_mappings.tsv
+```
+
+## Create the final table for manual curation
+```bash
+python3 bin/trait_mapping/create_table_for_manual_curation.py \
+  --traits-for-curation traits_requiring_curation.tsv \
+  --previous-mappings previous_mappings.tsv \
+  --output table_for_manual_curation.tsv
+```
+
+## Sort and export to Google Sheets
+Note that the number of columns in the output table is limited to 50, because only a few traits have that many
+mappings, and in virtually all cases these mappings are not meaningful. However, having a very large table degrades
+the performance of Google Sheets substantially.
+
+```bash
+cut -f-50 table_for_manual_curation.tsv | sort -t$'\t' -k2,2rn | xclip -selection clipboard
```

Comment:
make clearer that this won't work in the cluster (xclip)

Reply:
This is a good point! Actually I think usage of `xclip` here introduces more confusion than it actually helps. I rewrote this section to just indicate that you need to paste the contents of the file to Google Sheets table.